make/migrations/postgresql/0012_1.10.0_schema.up.sql

```diff
@@ -23,8 +23,9 @@ CREATE TABLE scan_report
     digest VARCHAR(256) NOT NULL,
     registration_uuid VARCHAR(64) NOT NULL,
     mime_type VARCHAR(256) NOT NULL,
-    job_id VARCHAR(32),
-    status VARCHAR(16) NOT NULL,
+    job_id VARCHAR(64),
+    track_id VARCHAR(64),
+    status VARCHAR(1024) NOT NULL,
```

Comment:
There are two migration sql files for 1.10: 0011 and 0015, how about merging them into one?

Reply:
Agree, we can do it in a new PR